### ISSUE TITLE
Improve docs for QNM sur

### ISSUE THREAD

**Issue by vijayvarma392** [vijayvarma392/surfinBH#29]
I should have caught this before merging the PR, but was held up with other things.
I noticed a few things that should be improved:

- [ ] The notebook probably needs to be rerun to remove old TODOs:
<img width="820" alt="image" src="https://github.com/user-attachments/assets/84b0b3eb-5726-470d-9950-da6f67038b95">

- [ ] (l,m,n,p) need to properly defined. For example, the allowed values of p are probably just (+1, -1). That and what that means should be stated. 

- [ ] Also, qnm conventions are different in different literature, so we should either point to the section in the paper where this is made clear, or repeat that here. For example, being an NR person, I might expect z-axis is always the direction of initial Lhat. Some ringdown people use zhat as the direction of final spin. These two are not the same when the remnant spin is negative. So, which one we use should be made clear in the docs.

**Comment by duetosymmetry** [comment]
No, they are not deps for getting these fits from surfinBH. But since the example notebook — which is converted with nbconvert when running tests — imports sxs and qnm, running the tests requires sxs and qnm.

**Comment by vijayvarma392** [comment]
> No, they are not deps for getting these fits from surfinBH. But since the example notebook — which is converted with nbconvert when running tests — imports sxs and qnm, running the tests requires sxs and qnm.

Ah, all good then, thanks!
